cwms-data-api/src/main/java/cwms/cda/data/dao/AuthDao.java

```diff
@@ -52,7 +61,11 @@ public class AuthDao extends Dao<DataApiPrincipal>{
     private static final String CHECK_API_KEY =
         "select userid from cwms_20.at_api_keys where apikey = ?";
 
-
+    public static final String CREATE_API_KEY = "insert into cwms_20.at_api_keys(userid,key_name,apikey,created,expires) values(UPPER(?),?,?,?,?)";
+    public static final String REMOVE_API_KEY = "delete from cwms_20.at_api_keys where UPPER(userid) = UPPER(?) and key_name = ?";
```

Comment:
If AT_API_KEYS table has an index, does it have a function-based index on UPPER(userid)?

Reply:
Userid is ALWAYS already uppercase. So i might not need the function here but it seemed clearer... and it's possible that will change in the future.